packages/slate-editor/src/modules/editor-v4-story-bookmark/components/StoryBookmarkElement/StoryBookmarkElement.tsx

```diff
@@ -52,51 +52,77 @@ export function StoryBookmarkElement({ attributes, children, element, params }:
         }
     }, [error]);
 
+    const hasStory = Boolean(!loading && story);
+
     return (
         <EditorBlock
             {...attributes} // contains `ref`
             element={element}
-            overlay="always"
-            renderMenu={({ onClose }) =>
-                story && (
-                    <StoryBookmarkMenu
-                        onClose={onClose}
-                        element={element}
-                        story={story}
-                        withNewTabOption={params.withNewTabOption}
-                        onUpdate={(attrs) => updateImage(editor, attrs)}
-                        onRemove={remove}
-                    />
-                )
+            overlay={hasStory ? 'always' : false}
+            renderMenu={
+                hasStory
+                    ? ({ onClose }) =>
+                          story && (
```

Comment:
I can see that there's another condition for menu rendering within the render function: `story && (`.

Can it not be combined with the `hasStory` condition to have less indents?